iam.tf

```diff
@@ -7,27 +7,27 @@ data "aws_iam_policy_document" "assume_role" {
 
     principals {
       type        = "Service"
-      identifiers = ["${slice(list("lambda.amazonaws.com", "edgelambda.amazonaws.com"), 0, var.lambda_at_edge ? 2 : 1)}"]
+      identifiers = slice(list("lambda.amazonaws.com", "edgelambda.amazonaws.com"), 0, var.lambda_at_edge ? 2 : 1)
```

Comment:
How far do you want to go to use new language features? This logic might become more straightforward, now that the ternary operator can return any type... https://www.terraform.io/docs/configuration/expressions.html#conditional-expressions

Reply:
Cool, that would be an improvement. I'll leave it as-is for now though, because I don't want to spend more time testing it.

Reply:
Ok, I'll turn the comment into an issue